Dockerfile

```diff
@@ -0,0 +1,41 @@
+ARG NGINX_VERSION=1.21
+
+# Stage 1 -- install dev tools and build bundle
+FROM node:18-bullseye-slim AS build
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  git \
+  ca-certificates \
+  && rm -rf /var/lib/apt/lists/*
+
+ARG UI_VERSION=latest
+
+COPY package-lock.json package.json ./
+COPY .env.production.template ./.env.production
+COPY ./scripts/replace-envvars.sh ./replace-envvars.sh
```

Comment:
this is probably not used here, so you could copy it directly from the host to the production image without this in-between step